frontend/src/features/business/components/VendorDetails.tsx

```diff
@@ -340,50 +387,50 @@ export function VendorDetails({ vendor, nearbyVendors }: VendorDetailsProps) {
                   )}
                 </Paper>
               </Box>
-            </Grid>
-
-            {/* Right Column - Contact Card */}
-            {vendor.email && (
-              <Grid size={{ xs: 12, md: 4 }}>
-                <StickyCard elevation={0} >
-                  <CardContent sx={{ p: 4 }}>
-                    <Typography variant="h5" component="h2" sx={{
-                      mb: 3,
-                      textAlign: 'center'
-                    }}>
-                      Get in Touch
-                    </Typography>
-                    <Button
-                      variant="contained"
-                      size="large"
-                      fullWidth
-                      startIcon={<Mail />}
-                      sx={{ mb: 3 }}
-                      href={`mailto:${vendor.email}`}
```

Comment:
We can link to the instagram if there's no email! Looks a little awkward without this button